### PR TITLE
Check if player is crouching

### PR DESCRIPTION
**Pull request by svr333**
## Description
Only activate the Lumber Axe when the player is crouching. (similar to explosivetools)

## Proposed changes
Add a crouch check in the LumberAxe

## Related Issues (if applicable)
[Discord issue 1515](https://discord.com/channels/565557184348422174/627048923122499584/892393743133577287)

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.14.* - 1.19.*).
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/LumberAxe.java [hunk 1]
```diff
@@ -67,6 +67,10 @@ private ToolUseHandler onBlockBreak() {
     public ItemUseHandler onItemUse() {
         return e -> {
             if (e.getClickedBlock().isPresent()) {
+                if (e.getPlayer().isSneaking()) {
```
Comment (Sefiraat) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/LumberAxe.java:70]: Would this not be better combined with the previous if statement? Inverted of course.

Comment (svr333) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/tools/LumberAxe.java:70]: oh yeah, definitely, looked over it

**Review by Sefiraat — APPROVED** [review]
👍

**Review by Sfiguz7 — APPROVED** [review]
LGTM

**Review by JustAHuman-xD — APPROVED** [review]
LGTM

**Review by Sefiraat — APPROVED** [review]
👍

**Review by JustAHuman-xD — APPROVED** [review]
LGTM v2.0

**Comment by Boomer-1** [comment]
did this then break the lumber axes ability to strip wood when shifting?

**Comment by Boomer-1** [comment]
oh wait ,that's right clicking. nevermind

**Comment by JustAHuman-xD** [comment]
> did this then break the lumber axes ability to strip wood when shifting?
> 
> 

I don't believe so

**Pull request merged**